tests/transforms/test_classical_jacobian.py

```diff
@@ -176,7 +176,7 @@ def test_tf_without_argnum(circuit, args, expected_jac, diff_method):
 def test_torch_without_argnum(circuit, args, expected_jac, diff_method):
     r"""Test ``classical_jacobian`` with ``argnum=None`` and Torch."""
     torch = pytest.importorskip("torch")
-    args = tuple((torch.tensor(arg) for arg in args))
+    args = tuple((torch.tensor(arg, requires_grad=True) for arg in args))
```

Comment:
Ah!

Reply:
Yes, indeed :D The tests should have been failing but weren't because of the bug :sweat_smile: